tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -1053,10 +1062,11 @@ def test_positive_end_to_end(self):
                 })
                 break
         # step 2.15.1: Enable product content
-        activation_key.content_override(data={'content_override': {
-            u'content_label': AK_CONTENT_LABEL,
-            u'value': u'1',
-        }})
+        if setting_is_set('fake_manifest'):
```

Comment:
@elyezer **setting_is_set** is called many times, and it has a getattr and also a call to **validate**, I know it is not too much expensive, but do you think it is a bad idea to call it only one time per modulle at the top?

```python
FAKE_MANIFEST_IS_SET = setting_is_set('fake_manifest')
```